app/models/wakes/resource.rb

```diff
@@ -28,6 +28,11 @@ def to_s
     EOS
   end
 
+  def recalculate_facebook_count!
```

Comment:
Hmmmmm.

I suppose that if you attached an existing `Wakes::Location` to a different `Wakes::Resource`, which is what the `Wakes.redirect` method does sometimes, then you're right, this becomes useful. I do not believe that my GA code takes that into account.

Leave this as a separate method. I will add a card to augment `Wakes.redirect` to do that for both FB and GA.